Dockerfile.template

```diff
@@ -34,7 +29,7 @@ EXPOSE 443
 ENV S6_LOGGING 0
 ENV S6_KEEP_ENV 1
 ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2
-ENV FTL_CMD no-daemon
+ENV FTL_CMD debug
```

Comment:
@pi-hole/api-approvers FYI no-daemon wasn't staying in the foreground for me in my hasty update last night, so I changed it to debug as a workaround to get things running (s6-service requires a foregrounded process)

Reply:
Thanks, I found the issue: pi-hole/FTL#430

Reply:
I just tried to run it as `pihole-FTL no-daemon` and - as @Mcat12 commented in his PR - observed that Ctrl+C, etc. are not caught anymore, however, `pihole-FTL` indeed stays in teh foreground and works as expected. We should continue this here: https://github.com/pi-hole/FTL/issues/432